xfails.txt

```diff
@@ -1,3 +1,8 @@
+# segmentation fault: https://github.com/onnx/onnx/pull/6570
+array_api_tests/test_manipulation_functions.py::test_repeat
+# segmentation fault: due to onnxruntime not handling einsum with dim 0
+array_api_tests/test_linalg.py::test_tensordot
```

Comment:
If these tests segfault then we should put them into skips.txt rather than here. That the array-api-tests are green nonetheless is an indication that something else is fishy. That feeling is confounded by the fact that they finish within 20s. Could you take a look, please?